src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -654,7 +654,17 @@ private static ImmutableArray<RefKind> GetEffectiveArgumentRefKinds(ImmutableArr
                 var receiverNamedType = invokedAsExtensionMethod ?
                                         ((MethodSymbol)methodOrIndexer).Parameters[0].Type.TypeSymbol as NamedTypeSymbol :
                                         methodOrIndexer.ContainingType;
-            isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+                isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+            }
+
+            var @params = methodOrIndexer.GetParameters();
```

Comment:
>var @params = methodOrIndexer.GetParameters(); [](start = 12, length = 46)

Consider moving this above `if (!ignoreComReceiver)` so `@params` can be used in that `if` block.

Reply:
Changed the method to make more use of `@params`. Thanks!

Reply:
Actually doesn't seem quite right, I'm gonna move the declaration up but the IsDefaultOrEmpty check back to where it originally was after ignoreComReceiver